test/parallel/test-dgram-multicast-setIf.js

```diff
@@ -0,0 +1,31 @@
+'use strict';
+require('../common');
+const assert = require('assert');
+const dgram = require('dgram');
+const socket = dgram.createSocket('udp4');
+const socket2 = dgram.createSocket('udp4');
+let thrown = false;
+
+socket.bind(0);
+socket.on('listening', function() {
+  socket.setMulticastInterface('0.0.0.0');
+
+  //close the socket
+  socket.close();
+});
+
+socket2.bind(0);
+socket2.on('listening', function() {
+
```

Comment:
For the above 3, I was using: test-dgram-multicast-setTTL.js as a template for requirements and style as it is the nearest test. I will make these updates using test-http-response-statuscode.js as a template.